### PR TITLE
Changes `recorded` to `sampled`

### PR DESCRIPTION
**Pull request by danielkhan**
As discussed in #265

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

**Review by discostu105 — APPROVED** [review]

**Comment by discostu105** [comment]
This will partly conflict with (also valid) changes from @arminru in #288. @danielkhan can you incorporate those improvements directly in this PR (#288 already has approvals).

**Comment by danielkhan** [comment]
> This will partly conflict with (also valid) changes from @arminru in #288. @danielkhan can you incorporate those improvements directly in this PR (#288 already has approvals).

I commented on #288 as the whole section is redundant to the processing model. Feel free to add your thoughts there.

File: spec/20-http_header_format.md [hunk 1]
```diff
@@ -449,12 +449,12 @@ Here is the list of allowed mutations:
...
    typical mutation and should be considered a default.
-2. **Update `recorded`**. The value of `recorded` reflects the caller's recording
+2. **Update `sampled`**. The value of `sampled` reflects the caller's recording
    behavior: either trace data was dropped or may have been recorded out-of-band -
    this can be indicated by toggling the flag in both directions. This mutation
    gives the downstream tracer information about the likelihood its parent's
-   information was recorded. `parent-id` MUST be set to a new value with the 
-   `recorded` flag update. See details of the [`recorded` flag](#recorded-flag-00000001)
+   information was recorded. `parent-id` MUST be set to a new value with the
+   `sampled` flag update. See details of the [`sampled` flag](#recorded-flag-00000001)
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]:
```suggestion
   `sampled` flag update. See details of the [`sampled` flag](#sampled-flag-00000001)
```

**Comment by SergeyKanzhelev** [comment]
in this PR I think this instance of word `recorded` should be replaced to `sampled`:

```
Value = 00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01
base16(version) = 00
base16(trace-id) = 4bf92f3577b34da6a3ce929d0e0e4736
base16(parent-id) = 00f067aa0ba902b7
base16(trace-flags) = 01  // recorded
```

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
Also change in security section https://w3c.github.io/trace-context/#security-considerations

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by nikmd23 — APPROVED** [review]

**Pull request merged**